src/resources/inside-flutter.md

```diff
@@ -328,6 +331,258 @@ for efficiently updating elements during scrolling and for modifying
 the render tree when elements are scrolled into and out of view at
 the edge of the viewport.
 
+# API Ergonomics
```

Comment:
The earlier section uses \` for class names, but this section doesn't (e.g. "Widget", "Element"). Is that intentional?

Reply:
No, I'll fix. It's a misunderstanding between me and our contractor.